docs/release_instructions.md

```diff
@@ -322,10 +322,6 @@ build the docker images locally, tag them with the release version
 dockerhub using the whiskbot dockerhub id, and update the `latest`
 tag to point to the new images.
 
-If you have published new images to dockerhub, submit PRs to
```

Comment:
This is repeated as the first paragraph of the very next section in the document.  Don't need it twice